ingest/settings.py

```diff
@@ -0,0 +1,64 @@
+# File is responsible for defining globals and initializing them
+from mongo_connection import MongoConnection
+from bson.objectid import ObjectId
+
+MONGO_CONNECTION = MongoConnection()
+
+
+def init(study_id, study_file_id):
+    global study
+    study = Study(study_id)
+    global study_file
+    study_file = StudyFile(study_file_id)
+
+
+def get_study():
+    global study
+    return study
+
+
+def get_study_file():
+    global study_file
+    return study_file
+
+
+class Study:
+    STUDY_ACCESSION: str = None
+
+    def __init__(self, study_id):
+        Study.update_study_accession(study_id)
+
+    @classmethod
+    def update_file_type(cls, file_type):
+        cls.FILE_TYPE = file_type
+
+    @classmethod
+    def update_study_accession(cls, study_id):
+        study_accession = list(
+            MONGO_CONNECTION._client["study_accessions"].find(
+                {"study_id": ObjectId(study_id)}, {"accession": 1, "_id": 0}
+            )
+        ).pop()
+        cls.STUDY_ACCESSION = study_accession["accession"]
+
+
+class StudyFile:
+    FILE_TYPE: str = None
+    FILE_SIZE: int = None
+    STUDY_FILE_ID = None
+    FILE_NAME = None
+
+    def __init__(self, study_file_id):
+        StudyFile.update(study_file_id)
+
+    @classmethod
+    def update(cls, study_file_id):
+        cls.STUDY_FILE_ID = study_file_id
+        query = MONGO_CONNECTION._client["study_files"].find(
+            {"_id": ObjectId("5e3dd9bcfa6429263aab24ba")},
+            {"upload_file_size": 1, "file_type": 1, "name": 1, "_id": 0},
+        )
+        query_results = list(query).pop()
+        cls.FILE_TYPE = query_results["file_type"]
+        cls.FILE_SIZE = query_results["upload_file_size"]
+        cls.FILE_NAME = query_results["name"]
```

Comment:
Like `study_accession`, passing these in as arguments from Rails to `ingest_pipeline.py` seems better than querying them from here.

Reply:
I dunno, Ingest Pipeline already has to know the intricacies of our study file/data array data models.  So I like keeping the interface between portal_core and pipeline simple -- passing in more and more args makes command-line testing of ingest pipeline more difficult

Reply:
You're right that the interface is simpler this way, but that also makes the pipeline more complex to test and maintain.  

Because we can't test database querying in Ingest Pipeline itself (due to lacking a MongoDB emulator, SCP-2000), the current approach means bugs in querying won't get caught until portal_core integration tests, at best.  Even after SCP-2000, this way entails more involved mocks to get the code covered.

By contrast, passing these as arguments lets us test them like other arguments, and means less code overall.

I defer to Eno.  This approach is reasonable, if not my ideal, and at least improves the case for a MongoDB emulator!